src/components/Flowbite/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { FC, PropsWithChildren, useLayoutEffect, useMemo } from 'react';
+import useDarkMode from '../../hooks/useDarkMode';
+import { ThemeContext } from '../../contexts/ThemeContext';
+
+export type FlowbiteProps = PropsWithChildren<{
+  children: any;
+  dark?: boolean;
+  usePreferences?: boolean;
```

Comment:
I think we should group the above two props because they are both related to the theme context.

Reply:
What would the benefits be?

Reply:
Later on, if we add more props to this component it might become unclear what every prop is doing.